source/elements/oneDAL/source/algorithms/decomposition/pca.rst

```diff
@@ -1,3 +1,447 @@
+.. highlight:: cpp
+.. default-domain:: cpp
+
 ===================================
 Principal Components Analysis (PCA)
 ===================================
+
+Principal Component Analysis (PCA) is an algorithm for exploratory data analysis
```

Comment:
why more minor thing to decide-we use capitalized terms to specify algorithm. Fine with me, but we need explicit simple rule contributors to the specs would be familiar with. And what other oneapi elements do from this perspective?